src/webassembly/instance.rs

```diff
@@ -551,20 +551,36 @@ impl Instance {
                 let memalign_export = module.info.exports.get("_memalign");
                 let memset_export = module.info.exports.get("_memset");
 
-                if let (Some(Export::Function(malloc_index)), Some(Export::Function(free_index)), Some(Export::Function(memalign_index)), Some(Export::Function(memset_index))) = (malloc_export, free_export, memalign_export, memset_export) {
-                    let malloc_addr = get_function_addr(&malloc_index, &import_functions, &functions);
-                    let free_addr = get_function_addr(&free_index, &import_functions, &functions);
-                    let memalign_addr = get_function_addr(&memalign_index, &import_functions, &functions);
-                    let memset_addr = get_function_addr(&memset_index, &import_functions, &functions);
-                    
+                let mut malloc_addr = 0 as *const u8;
```

Comment:
I noticed that for some simple C files, Em doesn't bundle a `mem_align` function. Could be an optimization thing. 
I changed the code to support when one or some of the functions are not bundled.

Reply:
But it doesn't support it, it'll just crash when it tries to call one of them.

Reply:
In my case, the C file I compiled doesn't contain a `mem_align`, but it contains others. 
I don't think any of them are being called anywhere because there is no malloc/free code in it.

Reply:
Do you have alternative solution to this.

Reply:
The file doesn't contain malloc or free?

Reply:
Oh, that's fine then. If the file doesn't have memalign, we can just write our own. Should be simple.